components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/LocalAuthenticatorConfig.java

```diff
@@ -194,4 +207,20 @@ public boolean equals(Object o) {
     public int hashCode() {
         return name != null ? name.hashCode() : 0;
     }
+
+    /**
+     * @return tags
```

Comment:
Let's add the method comments properly. Check all the new methods.

Reply:
Addressed with commit ad110cdff4ffcfe5766b31f1a09bebaf633730e6